packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "litefarm-api",
-  "version": "3.2.5",
+  "version": "3.2.6",
```

Comment:
this will have issue with https://github.com/LiteFarmOrg/LiteFarm/pull/2618  wich is the official PR for this change

Reply:
fixed thanks